test/e2e/storage/csi_mock_volume.go

```diff
@@ -1165,6 +1169,68 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 			})
 		}
 	})
+
+	ginkgo.Context("CSIServiceAccountToken [Feature:CSIServiceAccountToken]", func() {
```

Comment:
For alpha feature tests, we'll need to update test-infra alpha job to run this.

Reply:
https://github.com/kubernetes/test-infra/pull/19571

Reply:
PR merged